docs/user/guides/setup/git-repo-automatic.rst

```diff
@@ -1,27 +1,54 @@
-How to connect your Git repository
-==================================
+How to automatically configure a Git repository
+===============================================
 
 In this article, we explain how connecting your Read the Docs account to |git_providers_or|
-automatically configures your Git repository and your Read the Docs project.
+makes Read the Docs able to automatically configure your imported Git repositories and your Read the Docs projects.
 
 ✅️ Signed up with your Git provider?
   If you signed up or logged in to Read the Docs with your |git_providers_or|
   credentials, you're all done. Your account is connected.
-  The permissions that are granted are :ref:`explained below <connected-accounts:Permissions for connected accounts>`.
-⬇️ Signed up with your email address?
+
+  The rest of this guide helps to understand how the automatic configuration works.
+
+⏩️️ Signed up with your email address?
   If you have signed up to Read the Docs with your email address,
   you can add the connection to the Git provider afterwards.
-  See: :doc:`/guides/connecting-git-account`
+  You can also add a connection to an additional Git provider this way.
+
+  Please follow :doc:`/guides/connecting-git-account` in this case.
+
+Automatic configuration
+-----------------------
+
+When your Read the Docs account is connected to your Git Provider and you import a Git repository,
+all the basic settings will automatically be configured on the Read the Docs project and on your Git repository.
```

Comment:
I don't know what this means? What basic settings are configured?

Reply:
It seems that I have gone a bit over-the-top here, writing both "configured" and "settings".

New text:

> When your Read the Docs account is connected to |git_providers_or| and you import a Git repository,
> the integration will automatically be configured on the Read the Docs project and on your Git repository.